### PR TITLE
override Identifier for BufferedStream

### PR DESCRIPTION
**Pull request by amosonn**
## IMPORTANT 

* [x] The code complies with the [Coding Guidelines for C#](https://www.csharpcodingguidelines.com/).
* [ ] The changes are covered by a new or existing set of unit tests which follow the Arrange-Act-Assert syntax such as is used [in this example](https://github.com/fluentassertions/fluentassertions/blob/daaf35b9b59b622c96d0c034e8972a020b2bee55/Tests/FluentAssertions.Shared.Specs/BasicEquivalencySpecs.cs#L33).
* [x] ~~If the contribution adds a feature or fixes a bug, please update [**the release notes**](https://github.com/fluentassertions/fluentassertions/tree/master/docs), which are published on the [website](https://fluentassertions.com/releases).~~
* [x] ~~If the contribution changes the public API the changes needs to be included by running `AcceptApiChanges.ps1`/`AcceptApiChanges.sh`.~~
* [x] ~~If the contribution affects [the documentation](https://github.com/fluentassertions/fluentassertions/tree/develop/docs/_pages), please include your changes in this pull request so the documentation will appear on the [website](https://www.fluentassertions.com/introduction).~~

### CODE REVIEW

**Review by dennisdoomen — APPROVED** [review]

**Review by jnyrup — APPROVED** [review]

**Comment by amosonn** [comment]
Yeah, I didn't think this would affect the API (since that class inherits the property anyway). I will run the AcceptApiChanges.sh and push again.

File: Src/FluentAssertions/Streams/BufferedStreamAssertions.cs [hunk 1]
```diff
@@ -27,6 +27,8 @@ public BufferedStreamAssertions(BufferedStream stream)
         {
         }
 
+        protected override string Identifier => "bufferedstream";
```
Comment (jn-foreflight) [inline Src/FluentAssertions/Streams/BufferedStreamAssertions.cs:30]: I just had an extra look at how we do this elsewhere and as `Identifier` is included in a sentence it should be
```suggestion
        protected override string Identifier => "buffered stream";
```

Comment (jnyrup) [inline Src/FluentAssertions/Streams/BufferedStreamAssertions.cs:30]: I just had an extra look at how we do this elsewhere and as `Identifier` is included in a sentence it should be
```suggestion
        protected override string Identifier => "buffered stream";
```

Comment (amosonn) [reply Src/FluentAssertions/Streams/BufferedStreamAssertions.cs]: Changed this.

**Comment by jnyrup** [comment]
Thanks for contributing this fix 👍

**Pull request merged**